sigs.yaml

```diff
@@ -28,5 +28,29 @@ workinggroups:
   contact:
     slack: kubernetes-operators
     mailing_list: https://groups.google.com/forum/#!forum/operator-framework-wg-ci
+- dir: wg-olm
+  name: OLM
+  mission_statement: >
+    To organize community input for the Operator-Lifecycle-Manager project.
```

Comment:
Should we include the operator registry and opm in this as well?

Reply:
That's a good idea.

Reply:
Resolved in the latest version of the PR.